CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ### 3.0.1 (Next)
 
+* [#71](https://github.com/alexa-js/alexa-app-server/pull/71): Fixed [#68](https://github.com/alexa-js/alexa-app-server/issues/68), added utility functions, more descriptive error messages - [@tejashah88](https://github.com/tejashah88).
```

Comment:
What's important in changelog is to describe the actual fix to be helpful for the person reading it. So for example:

```
* [#71](https://github.com/alexa-js/alexa-app-server/pull/71), [#68](https://github.com/alexa-js/alexa-app-server/issues/68): Fixed log output containing multiple slashes - [@tejashah88](https://github.com/tejashah88).
```